stable/coredns/templates/deployment.yaml

```diff
@@ -36,8 +36,8 @@ spec:
         scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
         {{- end }}
     spec:
-      {{- if or .Values.rbac.create .Values.rbac.serviceAccountName }}
-      serviceAccountName: {{ if .Values.rbac.create }}{{ template "coredns.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
+      {{- if or .Values.serviceAccount.create .Values.serviceAccount.name }}
+      serviceAccountName: {{ if .Values.serviceAccount.create }}{{ template "coredns.fullname" . }}{{ else }}"{{ .Values.serviceAccount.name }}"{{ end }}
```

Comment:
reuse the template at https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md